docs/root/operations/admin.rst

```diff
@@ -164,3 +164,12 @@ The fields are:
   Outputs /stats in `Prometheus <https://prometheus.io/docs/instrumenting/exposition_formats/>`_
   v0.0.4 format. This can be used to integrate with a Prometheus server. Currently, only counters and
   gauges are outputed. Histograms will be outputed in a future update.
+
+.. http:get:: /runtime
+
+  Outputs all runtime values on demand. See :ref:`here <config_runtime_v1>` for more information on
```

Comment:
Maybe make explicit this is human readable text format (to contrast with JSON below).